### ISSUE TITLE
Why couldn't i download from Indavideo/Indavideo embedded if it is supported? HTTP Error 403

### ISSUE THREAD

**Issue by balazscseh** [ytdl-org/youtube-dl#32667]
## Checklist

- [X] I'm asking a question
- [X] I've looked through the README and FAQ for similar questions
- [X] I've searched the bugtracker for similar questions including closed ones

## Question

When i tried downloading from Indavideo or Indavideo embedded (saw support for it) i got the following error:
`https://indavideo.hu/video/Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip`
```
youtube-dl -v https://indavideo.hu/video/Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://indavideo.hu/video/Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, pref UTF-8
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.10.12 (CPython) - Linux-5.15.133.1-microsoft-standard-WSL2-x86_64-with-glibc2.35
[debug] exe versions: ffmpeg 4.4.2, ffprobe 4.4.2, rtmpdump 2.4
[debug] Proxy map: {}
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Requesting header
WARNING: Falling back on generic information extractor.
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Downloading webpage
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Extracting information
[download] Downloading playlist: Horseland - A Lovasklub - 1. Évad - 1. Rész - Nem A Kocsi Teszi A Lányt (Magyar Szinkron - DVDRip)
[generic] playlist Horseland - A Lovasklub - 1. Évad - 1. Rész - Nem A Kocsi Teszi A Lányt (Magyar Szinkron - DVDRip): Collected 1 video ids (downloading 1 of them)
[download] Downloading video 1 of 1
[IndavideoEmbed] cfe96d114d: Downloading JSON metadata
[debug] Default format spec: bestvideo+bestaudio/best
[debug] Invoking downloader on 'https://index1-hu.indavideo.hu/indavideo/002/571/846/Horseland.2006.S01E01.DVDRip.x264.Hun-Bluedog.360.mp4?channel=main-c&zone=hu&zone=upc&token=6f397a904cd1296069ef768c58e37414'
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/YoutubeDL.py", line 1976, in process_info
    success = dl(filename, info_dict)
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/YoutubeDL.py", line 1915, in dl
    return fd.download(name, info)
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/downloader/common.py", line 366, in download
    return self.real_download(filename, info_dict)
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/downloader/http.py", line 351, in real_download
    establish_connection()
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/downloader/http.py", line 116, in establish_connection
    raise err
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/downloader/http.py", line 110, in establish_connection
    ctx.data = self.ydl.urlopen(request)
  File "/usr/local/lib/python3.10/dist-packages/youtube_dl/YoutubeDL.py", line 2288, in urlopen
    return self._opener.open(req, timeout=self._socket_timeout)
  File "/usr/lib/python3.10/urllib/request.py", line 525, in open
    response = meth(req, response)
  File "/usr/lib/python3.10/urllib/request.py", line 634, in http_response
    response = self.parent.error(
  File "/usr/lib/python3.10/urllib/request.py", line 563, in error
    return self._call_chain(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 496, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.10/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

also when trying with purely embedded:

```
youtube-dl embed.indavideo.hu/player/video/cfe96d114d
WARNING: The url doesn't specify the protocol, trying with http
[IndavideoEmbed] cfe96d114d: Downloading JSON metadata
ERROR: unable to download video data: HTTP Error 403: Forbidden
```

**Comment by dirkf** [comment]
We're always subject to whatever changes the site decides to make, so "supported" refers to when the site was last tested. And I find that the media link can be extracted, with  a recent nightly build:
```console
dl
$ youtube-dl-20231123 -vF 'https://indavideo.hu/video/Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip'
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: [u'-vF', u'https://indavideo.hu/video/Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip']
[debug] Encodings: locale UTF-8, fs UTF-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2023.11.23 [aaed4884e] (single file build)
[debug] ** This version was built from the latest master code at https://github.com/ytdl-org/youtube-dl.
[debug] ** For support, visit the main site.
[debug] Python 2.7.15 (CPython i686 32bit) - Linux-6.1.0-13-686-pae-i686-with-debian-12.4 - OpenSSL 1.1.1a  20 Nov 2018 - glibc 2.1.3
[debug] exe versions: ffmpeg 5.1.4-0, ffprobe 5.1.4-0
[debug] Proxy map: {}
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Requesting header
WARNING: Falling back on generic information extractor.
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Downloading webpage
[generic] Horseland_-_A_Lovasklub_-_1_Evad_-_1_Resz_-_Nem_A_Kocsi_Teszi_A_Lanyt_Magyar_Szinkron_-_DVDRip: Extracting information
[download] Downloading playlist: Horseland - A Lovasklub - 1. Évad - 1. Rész - Nem A Kocsi Teszi A Lányt (Magyar Szinkron - DVDRip)
[generic] playlist Horseland - A Lovasklub - 1. Évad - 1. Rész - Nem A Kocsi Teszi A Lányt (Magyar Szinkron - DVDRip): Collected 1 video ids (downloading 1 of them)
[download] Downloading video 1 of 1
[IndavideoEmbed] cfe96d114d: Downloading JSON metadata
[info] Available formats for 2571846:
format code  extension  resolution note
0            mp4        360p       
[download] Finished downloading playlist: Horseland - A Lovasklub - 1. Évad - 1. Rész - Nem A Kocsi Teszi A Lányt (Magyar Szinkron - DVDRip)
$
```
However fetching the media gives 403. Maybe you need to be logged in (pass cookies from a logged-in browser session), or there is some restriction on your IP address (do you have to be in HU?).

Same result with _yt-dlp_.

When checking this item on the Site Support template that you should have used

>[ ]  I've checked that all provided URLs are alive and playable in a browser

you can open the browser's devtools and copy the headers used to fetch the media link (filter for `.360.mp4`) and post the result (maybe check for any sensitive data that you wouldn't want to publish).

**Comment by balazscseh** [comment]
I didn't used a nightly build, mabe that was the problem. Thanks for your help, i switched to [yt-dlp](https://github.com/yt-dlp/yt-dlp) and now it works fine without cookies or chaning geo-location.
